koku/masu/external/downloader/ibm/ibm_report_downloader.py

```diff
@@ -0,0 +1,242 @@
+from masu.util.ibm import common as util
+from providers.ibm.provider import IBMProvider
+
+
+DATA_DIR = Config.TMP_DIR
+LOG = logging.getLogger(__name__)
+REPORTS_DIR = Config.INSIGHTS_LOCAL_REPORT_DIR
+
+
+class NoMoreReportPages(Exception):
+    pass
+
+
+def page_downloader_factory(credentials, data_source, month):
+    service_version = "v1"
+    service_name = "resource-instance-usage-reports"
+    try:
+        authenticator = IAMAuthenticator(credentials.get("iam_token", ""))
+        service = BaseService(service_url="https://enterprise.cloud.ibm.com", authenticator=authenticator)
+    except Exception as exp:
+        LOG.error(f"Failed setting a base service. Reason: {str(exp)}")
+        raise exp
+    headers = get_sdk_headers(
+        service_name=service_name,
+        service_version=service_version.upper(),
+        operation_id=f"get_{service_name.replace('-', '_')}",
+    )
+    params = dict(enterprise_id=data_source.get("enterprise_id", ""), month=month, recurse=True, format="csv")
+
+    def download_page(page=1):
+        params["page"] = page
+        request = service.prepare_request(
+            method="GET", url=f"/{service_version}/{service_name}", headers=headers, params=params
+        )
+        try:
+            response = service.send(request)
+            return response
+        except ApiException as apiExc:
+            if "Total pages" in apiExc.message:
+                raise NoMoreReportPages(apiExc.message)
+            raise apiExc
+
+    return download_page
+
+
+def writer_factory(full_path):
+    local = dict(include_header=False)
+
+    def write(text):
+        include_header = local["include_header"]
+        status = "no-write"
+        with open(full_path, "a") as file_writer:
+            for line in text.splitlines():
+                if "the end of report" in line:
+                    break
+                if "Total Pages" in line:
+                    status = "ignore-meta-data"
+                    continue
+                if status == "ignore-meta-data" and not include_header and len(line) == 0:
+                    local["include_header"] = True
+                    status = "write"
+                    continue
+                if status == "ignore-meta-data" and include_header and "Account ID" in line:
+                    status = "write"
+                    continue
+                if status == "write":
+                    file_writer.write(f"{line}\n")
+
+    return write
+
+
+def download_pages_from(page_downloader, writer, page):
+    while True:
+        try:
+            response = page_downloader(page)
+        except NoMoreReportPages:
+            return
+        writer(response.get_result().text)
+        page += 1
+
+
+def create_daily_archives(
+    request_id, account, provider_uuid, local_filename, full_file_path, manifest_id, start_date, context
+):
+    if settings.ENABLE_S3_ARCHIVING or settings.ENABLE_PARQUET_PROCESSING:
```

Comment:
Can you replace `settings.ENABLE_PARQUET_PROCESSING` with `enable_trino_processing()`. Here's an example: https://github.com/project-koku/koku/blob/9d893857401c996990e8fedfcc4b6cb6156e1301/koku/masu/external/downloader/gcp/gcp_report_downloader.py#L95